src/platform/libretro/libretro_ui.cpp

```diff
@@ -565,11 +565,8 @@ RETRO_API unsigned retro_api_version() {
  * Can be called at any time, even before retro_init(). */
 RETRO_API void retro_get_system_info(struct retro_system_info* info) {
 	memset(info, 0, sizeof(*info));
-	info->library_name = "EasyRPG Player";
-	#ifndef GIT_VERSION
-	#define GIT_VERSION ""
-	#endif
-	info->library_version = PLAYER_VERSION GIT_VERSION;
+	info->library_name = GAME_TITLE;
+	info->library_version = Version::GetVersionString(true, false).c_str();
```

Comment:
C-Api crap: This returns a pointer to a temporary and will crash.

Possible workaround:

```cpp
static std::string version = Version::GetVersionString(true, false);
info->library_version = version.c_str();
```

Pointers to ``.c_str()`` are also kinda unsafe but this works when the std::string is not modified afterwards.

Reply:
I figured it would not work :D 
Thank you for the fix.
Other solution would be replicating the definitions in cmake like for version.cpp.